### DISCUSSION TITLE
Cannot Update Model Name/Parameters

### DISCUSSION THREAD

**Discussion by reddev-aroy** [open-webui/open-webui#3719] [General]
I am currently facing issue to update a model name/parameters. When I click on Save&Update, I am getting below on terminal:
'Model' object has no attribute 'update'

I am currently on the latest changes of origin/dev using manual install; 

Ollama is running on a different machine and being served by local ipv4 in openwebui.

![image](https://github.com/open-webui/open-webui/assets/168908647/9d11af06-a632-465c-910a-dcfd102c7595)

**Comment by silentoplayz** [comment]
Thank you for reporting this issue. It has been documented internally and the Open WebUI team is actively working towards a resolution. We'll aim to have a fix available soon and updates will be pushed to the dev branch.

**Reply by reddev-aroy** [reply]
Sure, thanks for quick update!

**Reply by silentoplayz** [reply]
Should be fixed on the dev branch by now.
